config/default.ini

```diff
@@ -4,7 +4,7 @@ name:          Scratch2Catrobat Converter
 short_name:    S2CC
 version:       0.10.0
 build_name:    Aegean cat
-build_number:  995
+build_number:  997-STCC-200_explicit_python2
```

Comment:
default.ini build_number should not contain any litterals only numbers. This got fixed by another commit, please rebase